File: gm-utils/src/main/java/src/commom/utils/string/StringContains.java
package src.commom.utils.string;

import src.commom.utils.object.Null;

public class StringContains {

	private StringContains() {}

	public static boolean is(String a, String b) {
		if (Null.is(a) || Null.is(b)) {
			return false;
		}
		if (b.length() == 0) {
			return true;
		}
		if (a.length() == 0) {
			return false;
		}
		return a.indexOf(b) > -1;
	}
	
}

File: gm-utils/js/src/commom/utils/string/StringContains.js
import Null from '../object/Null';

export default class StringContains {

	static is(a, b) {
		if (Null.is(a) || Null.is(b)) {
			return false;
		}
		if (b.length === 0) {
			return true;
		}
		if (a.length === 0) {
			return false;
		}
		return a.indexOf(b) > -1;
	}

}
